clients/drcachesim/tools/view.cpp

```diff
@@ -424,9 +424,13 @@ view_t::parallel_shard_memref(void *shard_data, const memref_t &memref)
             std::cerr << "<marker: system call trace end>\n";
             break;
         case TRACE_MARKER_TYPE_BRANCH_TARGET:
+            // These are not expected to be visible but we handle nonetheless.
```

Comment:
Since it's added to memref.instr.indirect_branch_target and not visible separately.

Reply:
Yes, augmented comment.